experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -139,15 +141,11 @@ export interface DetachEvent {
 	 * The intention of edit which last emptied the cell.
 	 */
 	revision: RevisionTag;
-
-	/**
-	 * The absolute position of the node in this cell in the input context of the revision which emptied it.
-	 */
-	index: number;
+	id: ChangesetLocalId;
```

Comment:
Any reason not to start using ChangeAtomId instead of DetachEvent?

Reply:
No

Reply:
There are some places where ChangeAtomId allowing the revision to be undefined is a bit annoying.

Reply:
Ah, yeah, that's been annoying. Happy to keep DetachEvent as well.